AntWars/Board/Ants/Warrior.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AntWars.Board.Ants
+{
+    public class Warrior : Ant
+    {
+        public int AttackPower { get; internal set; }
```

Comment:
Sollte in der ``Ant.cs`` sein, auch wenn Scout und Carrier 0 hätten, sollten sie dieses Attribute haben.